pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_import_modules.adoc

```diff
@@ -0,0 +1,19 @@
+[id="import_modules_{context}"]
```

Comment:
the id should be similar to the module heading, I suggest changing the id to: 
[id="importing-modules-into-pantheon_{context}"]

Reply:
done, changed ID and file name to correspond